Gemfile

```diff
@@ -13,14 +13,23 @@ gem 'jbuilder', '~> 2.0'
 gem 'dotenv-rails'
 gem 'therubyracer'
 gem 'less-rails'
```

Comment:
Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `dotenv-rails` should appear before `jbuilder`.